frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -875,24 +878,28 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
             <dd>
               <ResourceLink kind={referenceForModel(ClusterVersionModel)} name={cv.metadata.name} />
             </dd>
-            <dt>{t('cluster-settings~Cluster autoscaler')}</dt>
-            <dd>
-              {_.isEmpty(autoscalers) ? (
-                <Link to={`${resourcePathFromModel(ClusterAutoscalerModel)}/~new`}>
-                  <AddCircleOIcon className="co-icon-space-r" />
-                  {t('cluster-settings~Create autoscaler')}
-                </Link>
-              ) : (
-                autoscalers.map((autoscaler) => (
-                  <div key={autoscaler.metadata.uid}>
-                    <ResourceLink
-                      kind={clusterAutoscalerReference}
-                      name={autoscaler.metadata.name}
-                    />
-                  </div>
-                ))
-              )}
-            </dd>
+            {clusterAutoscalerFlag && (
```

Comment:
I'd also adjust the list of resources we pass to `Firehose` so we don't try to fetch the autoscalers when the CRD is not there.

Reply:
Updated

Reply:
But I broke the unit test.  Hold a min.

Reply:
Unit test fixed.